### DISCUSSION TITLE
How do I reset the autosave and/or start from last game in files?

### DISCUSSION THREAD

**Discussion by DavidGHaydock** [OnionUI/Onion#1435] [Support]
I installed Drastic, tried to run a game and it crashed. I then turned the miyoo off and on forgetting that I had enabled "autostart from last save". and it has autosaved the crash. Now every time I start up its a black screen.

**Comment by Aemiii91** [comment]
> Hold MENU while turning on, then it will boot to menu.

https://github.com/OnionUI/Onion/discussions/1173#discussioncomment-7135233

You'll also need to either delete the save state file, or simply select "Reset game" in the GLO menu (press Y in the game list), this will start the game without the save state (falling back to the regular game save).

**Reply by DavidGHaydock** [reply]
Hi Aemiii, thank you for the response. Unfortunately the menu function doesn't work, it remains a black screen. I'll have a go at deleting the save files to see if that'll fix it. Could you point me to where the save files for drastic would be in the file structure? I can't seem to find them. Thank you again for your help!

**Reply by Aemiii91** [reply]
First try deleting the `.tmp_update/cmd_to_run.sh` file (that is responsible for auto-resume on startup)

Save states are located in: `Saves/CurrentProfile/states/[core]/`.

**Reply by DavidGHaydock** [reply]
There was no cmd_to_run.sh file, and there actually is no save folder in /states that is for the drastic core, which means that my understanding of what the problem was wrong. Any ideas on what to suggest here? Maybe just a reinstall or "factory reset"?

**Reply by Aemiii91** [reply]
Ah, I didn't take notice that is was drastic issues. If the system starts up to a black screen without a `cmd_to_run.sh` file, there must be something wrong with your system files, and you'll probably need to reinstall Onion.

**Reply by ambrox20** [reply]
@Aemiii91 I really appreciate that 90 hours in a ROM hack, I didn't want to lose all my progress. You are the best!!!
